### PR TITLE
fix(banner): Swap out check using onboardingTask with checking projects array

### PR DESCRIPTION
**Pull request by denamwangi**
Not sure why but there are cases where the `FirstEvent` `OrganizationOnboardingTask` does not get saved to the db but the org has at least one project with `FirstEvent`. This PR shifts the check to look through the org projects array instead while we dig into whatever is causing the issue with the `Task`.

### CODE REVIEW

File: src/sentry/static/sentry/app/components/installPromptBanner.jsx [hunk 1]
```diff
@@ -41,11 +40,11 @@ const InstallPromptBanner = createReactClass({
...
 
   sentFirstEvent() {
-    let {onboardingTasks} = this.props.organization;
-    let firstEventTask = onboardingTasks.find(task => task.task === 2);
-    this.setState({
-      sentFirstEvent: firstEventTask && firstEventTask.status === 'complete',
-    });
+    let {projects} = this.props.organization;
+    let hasFirstEvent = projects.find(
+      project => project.firstEvent && project.firstEvent !== null
```
Comment (billyvg) [inline src/sentry/static/sentry/app/components/installPromptBanner.jsx]: Is the null comparison necessary, anything truthy will never be equal to `null`

```suggestion
      project => project.firstEvent
```

File: src/sentry/static/sentry/app/components/installPromptBanner.jsx [hunk 2]
```diff
@@ -41,11 +40,11 @@ const InstallPromptBanner = createReactClass({
   },
 
   sentFirstEvent() {
-    let {onboardingTasks} = this.props.organization;
-    let firstEventTask = onboardingTasks.find(task => task.task === 2);
-    this.setState({
-      sentFirstEvent: firstEventTask && firstEventTask.status === 'complete',
-    });
+    let {projects} = this.props.organization;
+    let hasFirstEvent = projects.find(
```
Comment (billyvg) [inline src/sentry/static/sentry/app/components/installPromptBanner.jsx]: You could shorten this to 

```suggestion
    return projects.find(
```

or this to hint that you're expecting a boolean value to be returned

```suggestion
    return !!projects.find(
```

**Review by billyvg — COMMENTED** [review]
What happens if your org has closed membership and a project you don't have access to has a `firstEvent`, do we still want to show this banner?

**Comment by denamwangi** [comment]
We want to keep it org level so that if any project anywhere in the org has a first event then the banner should disappear for all members. It looks like `organization.projects` holds all projects whether i'm part of the team or not (is closed membership different/stricter than team membership? i.e. is it possible for the list of projects to be filtered down?)

**Review by billyvg — APPROVED** [review]
